sites/partners/src/components/listings/PaperListingDetails/sections/DetailNotes.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useContext } from "react"
+import { t, GridSection, GridCell, ExpandableText } from "@bloom-housing/ui-components"
+import { FieldValue } from "@bloom-housing/ui-seeds"
+import { ListingContext } from "../../ListingContext"
+import { getDetailFieldString, getDetailFieldDate } from "./helpers"
+import { ListingStatus } from "@bloom-housing/backend-core"
+
+const DetailListingNotes = () => {
+  const listing = useContext(ListingContext)
+
+  if (!listing.requestedChanges || !(listing.status === ListingStatus.changesRequested)) return
+
+  return (
+    <GridSection
+      className="bg-primary-lighter"
+      title={t("listings.approval.listingNotes")}
+      inset
+      columns={3}
+    >
+      <GridCell span={2}>
+        <FieldValue id="requestedChanges" label={t("listings.approval.changeRequestSummary")}>
+          <ExpandableText
+            buttonClassName="ml-4"
+            strings={{
+              readMore: t("t.more"),
+              readLess: t("t.less"),
+            }}
+          >
+            {getDetailFieldString(listing.requestedChanges)}
+          </ExpandableText>
+        </FieldValue>
+      </GridCell>
+
+      <GridCell span={2}>
+        <FieldValue id="requestedChangesDate" label={t("listings.approval.requestDate")}>
+          {getDetailFieldDate(listing.requestedChangesDate)}
```

Comment:
I noticed the figma has a different date format. But that is inconsistent with the other date fields on the page 
![image](https://github.com/bloom-housing/bloom/assets/42942267/95bb3da3-fe97-40d2-90e7-1ff0a0113379)

Also, this is more of a design question, but would it be useful to have the time as well? If there is a lot of back and forth on the same day it might be useful to have a timestamp

Reply:
Will ask product / design at our check-in today!

Reply:
Confirmed w product to keep the format in the PR